frontend/src/pages/node/AssociationsTable.vue

```diff
@@ -303,6 +306,10 @@ watch(
   () => props.category,
   async () => await queryAssociations(true),
 );
+watch(
+  () => props.includeOrthologs,
+  async () => await queryAssociations(true),
```

Comment:
Technically don't need the `async` and `await` here since you're not waiting for it to finish to do something afterward.